src/Microsoft.Tye.Hosting/TyeHost.cs

```diff
@@ -298,6 +306,31 @@ private async Task StopAsync()
             _processor = null;
         }
 
+        private void OpenDashboard(string url)
+        {
+            try
+            {
+                //still not addressed https://github.com/dotnet/corefx/issues/10361
+                if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+                {
+                    url = url.Replace("&", "^&");
+                    System.Diagnostics.Process.Start(new ProcessStartInfo("cmd", $"/c start {url}") { CreateNoWindow = true });
```

Comment:
Do we need quotes around these URLs?

Reply:
yes, powershell works fine. However if zsh or other shell add-ins. Quotes ensure success.